cpp/benchmarks/secir_ageres_setups.h

```diff
@@ -23,8 +23,8 @@
 #include "memilio/compartments/simulation.h"
 #include "memilio/mobility/mobility.h"
 #include "memilio/compartments/parameter_studies.h"
-#include "models/secir/secir.h"
-#include "models/secir/parameter_space.h"
+#include "models/ode_secir/secir.h"

```

Comment:
Please have a search for `secir.h` and `secir.cpp` which do not exist anymore (except an cpp example maybe)